webpack.config.node.js

```diff
@@ -105,6 +105,9 @@ var webpackConfig = {
 	plugins: [
 		// Require source-map-support at the top, so we get source maps for the bundle
 		new webpack.BannerPlugin( 'require( "source-map-support" ).install();', { raw: true, entryOnly: false } ),
+		new webpack.DefinePlugin( {
```

Comment:
I wouldn't expect it to work in `sections.js`, but it should work at run time in different environments, which I think is fine.